.github/workflows/build_helm_svc_defns.yml

```diff
@@ -5,8 +5,35 @@ on:
   workflow_dispatch:
 
 jobs:
+
+  file-check:
+    runs-on: self-hosted
+    outputs:
+      any-modified: ${{steps.check-file-output.outputs.any-modified}}
+    name: Check changed files
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v39
+        with:
+          files: |
+            Chart.yaml
+            services/**
+
+      - name: Set Output
```

Comment:
Why do we need this step @varenyatessell ; Can we not use `steps.changed-files.outputs.any_modified`

Reply:
Yes we can, this way it helps in better debugging since what output is coming is visible.